config.json

```diff
@@ -3,7 +3,8 @@
 		"listen_url": "127.0.0.1:3333",
 		"use_tls": true,
 		"cert_path": "gophish_admin.crt",
-		"key_path": "gophish_admin.key"
+		"key_path": "gophish_admin.key",
+		"trusted_origins": ""
```

Comment:
This should be:

```
"trusted_origins": []
```

to match the structure in `config/config.go`.

Reply:
oof

Yep, I should have handled that.

Reply:
https://github.com/gophish/gophish/pull/2301/commits/3759e78e7dcc3f3c9adaca1d16b286e3b961b63c

Reply:
:+1: